.editorconfig

```diff
@@ -0,0 +1,9 @@
+root=true
+
+[*]
+
+charset = utf-8
+indent_style = tab
+indent_size = 2
+trim_trailing_whitespace = true
+insert_final_newline = false
```

Comment:
What is this?

Reply:
so that we all have the same formatting

Reply:
Not a blocker, but the feature should only contain what is necessary, this file should be added in a separate PR

Reply:
noted